### PR TITLE
Bug-11 : Language does not change to french in the calendar component 

### PR DESCRIPTION
**Pull request by TalalBazerbachi**
Add translation to text in the calendar component

### CODE REVIEW

File: components/menu/calendar/Screens/FetchScreen/CheckboxGroup.js [hunk 1]
```diff
@@ -152,7 +153,7 @@ render() {
 
       <View style={styles.button}>
         <Button
-          title="Synchronize Calendars"
+          title={i18n.t('SynchronizeCalendars')}
```
Comment (Belize22) [inline components/menu/calendar/Screens/FetchScreen/CheckboxGroup.js]: violates camel case notation.

File: components/menu/calendar/Screens/DashboardScreen/index.js [hunk 2]
```diff
@@ -311,12 +329,14 @@ export default class DashboardScreen extends Component {
         >
           <DialogInput
             isDialogVisible={this.state.isDialogVisible}
-            title="Set Reminder Time"
+            title={i18n.t('setRemindarTime')}
```
Comment (Belize22) [inline components/menu/calendar/Screens/DashboardScreen/index.js]: Misspelling

File: localization-context.js [hunk 3]
```diff
@@ -42,7 +74,23 @@ const SetLocaleContext = () => {
       back: 'Atrás',
       LOCATION_ALERT_TITLE: 'Servicios de ubicación requeridos',
       LOCATION_ALERT_MESSAGE: 'Los servicios de ubicación no se han habilitado. Permita que esta aplicación use su información de ubicación.',
-      LOCATION_ALERT_BUTTON: 'Configuración abierta'
+      LOCATION_ALERT_BUTTON: 'Configuración abierta',
```
Comment (Belize22) [inline localization-context.js:79]: Good opportunity to convert these tags to camelCase

File: components/menu/calendar/Screens/DashboardScreen/index.js [hunk 4]
```diff
@@ -60,7 +60,12 @@ export default class DashboardScreen extends Component {
   registerForPushNotificationsAsync = async () => {
     const { status } = await Permissions.askAsync(Permissions.NOTIFICATIONS);
     if (status !== 'granted') {
-      alert('No notification permissions!');
+      // alert(i18n.t('permisionNotGrandted'));
```
Comment (Belize22) [inline components/menu/calendar/Screens/DashboardScreen/index.js]: Dead alert

File: components/menu/calendar/Screens/DashboardScreen/index.js [hunk 5]
```diff
@@ -60,7 +60,12 @@ export default class DashboardScreen extends Component {
...
+      Alert.alert(i18n.t('permisionNotGrandted'),
```
Comment (Belize22) [inline components/menu/calendar/Screens/DashboardScreen/index.js]: Misspelling

File: components/menu/calendar/Screens/DashboardScreen/index.js [hunk 6]
```diff
@@ -172,7 +177,12 @@ export default class DashboardScreen extends Component {
         }
       } else {
         // your call back function
-        alert('Please enter numbers only');
+        // alert(i18n.t('permisionNotGrandted'));
```
Comment (Belize22) [inline components/menu/calendar/Screens/DashboardScreen/index.js]: Dead Alert

File: components/menu/calendar/Screens/DashboardScreen/index.js [hunk 7]
```diff
@@ -311,12 +329,14 @@ export default class DashboardScreen extends Component {
         >
           <DialogInput
             isDialogVisible={this.state.isDialogVisible}
-            title="Set Reminder Time"
+            title={i18n.t('setRemindarTime')}
             keyboardType="numeric"
-            message="Set the time to be reminded before class/minutes"
+            message={i18n.t('remindarMessage')}
```
Comment (Belize22) [inline components/menu/calendar/Screens/DashboardScreen/index.js]: Misspelling

**Review by mohhef — APPROVED** [review]
Everything so far i see is translated, very nice job

**Review by Belize22 — CHANGES_REQUESTED** [review]
Address the mentioned code quality issues.

**Review by sebhani — APPROVED** [review]
Works on my phone.

**Comment by TalalBazerbachi** [comment]
@Belize22 changes have been addressed

**Review by eufekt — APPROVED** [review]

**Pull request merged**